conanfile.py

```diff
@@ -200,17 +210,24 @@ def get_configure_command_args(self):
         if not self.options.with_ldap:
             params.append("--disable-ldap")
 
-        if self.options.custom_cacert:
-            params.append('--with-ca-bundle=cacert.pem')
+        if self.options.with_ca_bundle == False:
+            params.append("--without-ca-bundle")
+        elif self.options.with_ca_bundle:
+            params.append(f"--with-ca-bundle={str(self.options.with_ca_bundle)}")
 
+        if self.options.with_ca_path == False:
+            params.append('--without-ca-path')
+        elif self.options.with_ca_path:
+            params.append(f"--with-ca-bundle={str(self.options.with_ca_bundle)}")
```

Comment:
The same f-syntax fix.
Also should not it be `with-ca-path` instead of `with-ca-bundle`?

```suggestion
            params.append("--with-ca-path=" + self.options.with_ca_path)
```